core/ipc/twin/common/src/main/java/org/opennms/core/ipc/twin/common/AbstractTwinSubscriber.java

```diff
@@ -163,6 +163,9 @@ private void validateAndHandleUpdate(TwinUpdate twinUpdate) {
     }
 
     private byte[] applyPatch(TwinTracker twinTracker, TwinUpdate twinUpdate) {
+        if (!twinUpdate.isPatch()) {
+            twinUpdate.getObject();
```

Comment:
This does nothing...

Reply:
when applying patch fails at publisher side and when we get an update with full object, this may come into picture.

Reply:
But it should `return`, right?

Reply:
err, resolved!